lerna.json

```diff
@@ -11,6 +11,12 @@
     }
   },
   "version": "9.2.8",
-  "npmClient": "yarn",
-  "useWorkspaces": true
+  "npmClient": "npm",
```

Comment:
this surprises me - the commit message say about using yarn, but here u are changing the npm client to, well, npm

Reply:
Yeah, lerna is only used for publishing and I want to use npm to publish because yarn doesn't respect publishConfig.tag and npm does.

Reply:
ok, makes sense - maybe add a note to some kind of Contributing / Developing section about this?